.github/renovate.json

```diff
@@ -51,7 +52,7 @@
     },
     {
       "matchDepTypes": ["devDependencies"],
-      "matchPackagePatterns": ["^jest-", "jest$", "^prettier", "prettier$"],
+      "matchPackagePatterns": ["^jest-", "jest$", "^prettier", "prettier$", "lint-staged"],
```

Comment:
Realized we could include this in addition.

Reply:
Yup, good spot!